### DISCUSSION TITLE
2025 season schedule update

### DISCUSSION THREAD

**Discussion by nirbhaykwatra** [theOehrly/Fast-F1#662] [General]
Hi! I wanted to ask if/when would the 2025 event schedule data be available using the get_event_schedule method?

**Comment by theOehrly** [comment]
The schedule for 2025 is not yet available on formula1.com. I assume that it is not yet finalized beyond which weekend is which race. And even that might be subject to change still. The schedule through FastF1 cannot be available before that. When the final schedule will be announced, I don't know.

**Reply by theOehrly** [reply]
@nirbhaykwatra the 2025 schedule is now available

**Reply by nirbhaykwatra** [reply]
Thanks so much!
